test/app.js

```
@@ -0,0 +1,134 @@
+                        timeStamp: currentTime,
+                        to: sms.to,
+                        from: sms.from,
+                        message: txt,
+                        submitTime: sms.submitTime,
+                        //smsCount: sms.pduObjs.length,
+                        apiUrl: url,
+                        charset: this.dta_charset,      // Character set
+                        encoding: this.data_coding
+                };
+
+                request(url, {timeout: 60000} ,function (error, response, body) {
```

ESLint

[error] Expected indentation of 2 tabs but found 16 spaces.

(indent)